code/game/machinery/computer/buildandrepair.dm

```diff
@@ -248,6 +248,13 @@
 	name = "circuit board (Library Visitor Console)"
 	build_path = /obj/machinery/computer/libraryconsole
 	origin_tech = "programming=1"
+/obj/item/weapon/circuitboard/bot_upgrade_boost
```

Comment:
It would still be possible to print it using the exosuit fabricator though, right?